packages/mobile/types/CountdownCircleTimer.d.ts

```diff
@@ -18,7 +18,7 @@ export interface CountdownCircleTimerProps {
   /** Countdown duration in seconds */
   duration: number
   /** Single color as a string or an array of tuples: 1st param - color in HEX format; 2nd param - time to transition to next color represented as a fraction of the total duration */
-  colors: string | Colors
+  colors: string[][] | Colors[][]
```

Comment:
Hey @OmranElmasry, please elaborate on the problem you are trying to fix with this PR. The proposed solution does not satisfy the criteria for the `color` props, which is described in the comment above the change you did.  

> /** Single color as a string or an array of tuples: 1st param - color in HEX format; 2nd param - time to transition to next color represented as a fraction of the total duration */